src/mlx90632.c

```diff
@@ -594,4 +594,97 @@ int32_t mlx90632_start_measurement_burst(void)
     return 0;
 }
 
+
+STATIC int32_t unlock_eeporm()
```

Comment:
please prefix these with mlx90632 to avoid clashing with user code (`unlock_eeprom` has a typo anyway)
```suggestion
STATIC int32_t mlx90632_unlock_eeprom()
```